docker/docker-compose.yml

```diff
@@ -22,7 +34,24 @@ services:
       timeout: 5s
       retries: 5
   postgres:
-    image: postgres:15
+    image: docker.io/library/postgres:15
+    cap_drop:
+      - ALL
+    cap_add:
+      - CHOWN
+      - DAC_READ_SEARCH
+      - FOWNER
+      - SETGID
+      - SETUID
+    security_opt:
+      - no-new-privileges:true
```

Comment:
`- no-new-privileges: true` (with space)

Reply:
@dtslvr unfortunately this particular field is not compatible with that style, similar to the volumes field: `- postgres:/var/lib/postgresql/data`